### DISCUSSION TITLE
Custom voice for selected language

### DISCUSSION THREAD

**Discussion by rtm-1dyakonov** [coqui-ai/TTS#2910] [General Q&A]
Hello everyone!

I am interested in converting text into spoken audio but with specific voice. 
There is already a code of my language case so I assume the real question is if there is a way to change the voice itself without affecting the language? If there isn't can I train my own model then? 

BTW I have a bunch of samples of the voice I'm interested in.

**Comment by thorstenMueller** [comment]
You can finetune an existing model with your individual samples. Changing an existing voice, using SSML is not (yet) supported. Or you can train a new model if you have "enough" samples available. I've made two tutorials on that.

* https://youtu.be/bJjzSo_fOS8
* https://youtu.be/4YT8WZT_x48
